components/user_settings/modal/user_settings_modal.jsx

```diff
@@ -110,10 +110,14 @@ class UserSettingsModal extends React.Component {
         document.removeEventListener('keydown', this.handleKeyDown);
     }
 
-    componentDidUpdate() {
+    componentDidUpdate(_prevProps, prevState) {
         if (!Utils.isMobile()) {
             $('.settings-content .minimize-settings').perfectScrollbar('update');
         }
+
+        if (this.state.active_tab !== prevState.active_tab) {
+            $(ReactDOM.findDOMNode(this.refs.modalBody)).scrollTop(0);
```

Comment:
Not for now, when is possible is way better to migrate to the new `React` approach using `createRef` for this kind of things, avoiding any `findDOMNode` need. But for a PR to fix a bug a day before of the release cut, works for me :)